spec/models/article_spec.rb

```diff
@@ -222,6 +222,15 @@ def build_and_validate_article(*args)
     end
 
     describe "title validation" do
+      it "removes extra spaces" do
+        # Rubocop objects to this for many reasons, but it's intentional
+        # rubocop:disable
```

Comment:
`rubocop:disable Layout/LineLength` perhaps?

Reply:
It had a couple different objections to this line so I just tried to disable all of them. It was hard to tell locally, though, because Rubocop was examining this entire file (not just the diff) and it objected to lots of things in the file. I assume new rules were added with Rubocop updates. But it was just a mess with stuff that had nothing to do with my PR, so I skipped the local linting entirely. 😕